table.cc

```diff
@@ -962,7 +962,9 @@ void table::set_metrics() {
                 ms::make_gauge("live_disk_space", ms::description("Live disk space used"), _stats.live_disk_space_used)(cf)(ks),
                 ms::make_gauge("total_disk_space", ms::description("Total disk space used"), _stats.total_disk_space_used)(cf)(ks),
                 ms::make_gauge("live_sstable", ms::description("Live sstable count"), _stats.live_sstable_count)(cf)(ks),
-                ms::make_gauge("pending_compaction", ms::description("Estimated number of compactions pending for this column family"), _stats.pending_compactions)(cf)(ks)
+                ms::make_gauge("pending_compaction", ms::description("Estimated number of compactions pending for this column family"), _stats.pending_compactions)(cf)(ks),
+                ms::make_queue_length("pending_sstable_deletions", [this] { return _sstable_deletion_sem.waiters(); },
```

Comment:
A thought about naming:
Maybe we should name all the queue length metrics in some common way so they will be easier to recognize as such? I saw you added one metric called "something_queue_length" and here "pending_something", for example.

Reply:
I tried to be consistent locally - e.g. when adding a queue length here, I used "pending", but in other metric groups you can also find "queued_something" or "something_current"... and changing existing metrics is out of the question, so there's not that much I can do. I think that the most important thing is to use `ms::make_queue_length` instead of this generic `ms::gauge` - then, when you fetch `localhost:9180/metrics`, this metric is already marked as "queue_length", which is very descriptive.